geode-core/src/main/java/org/apache/geode/internal/cache/versions/RegionVersionVector.java

```diff
@@ -606,10 +606,11 @@ public void recordVersion(T mbr, VersionTag<T> tag) {
       // recovers. So we can only assert that the local member has already seen
       // the replayed event.
       synchronized (localExceptions) {
-        if (this.localVersion.get() < tag.getRegionVersion()) {
+        if (region != null && region.isInitialized()
```

Comment:
Minor quibble, but since the normal flow is that the local version will not be less that the tag's, that check should go first and short-circuit checking anything about the region, which should only need to be checked on very rare occasions.

Reply:
Good catch...Fixed in new diff.